tfjs-backend-webgpu/src/setup_test.ts

```diff
@@ -124,6 +130,7 @@ const TEST_FILTERS: TestFilter[] = [
       'gradient'  // gradient function not found.
     ]
   },
+  {startsWith: 'isNaN ', excludes: []},
```

Comment:
You need to remove related tests from the last list.

Reply:
The isNaN op is still used.

Reply:
Thanks, done, PTAL